cmd/neofs-cli/modules/util/acl.go

```diff
@@ -140,7 +140,6 @@ func eaclFiltersToString(fs []eacl.Filter) string {
 			_, _ = tw.Write([]byte("\t==\t"))
 		case eacl.MatchStringNotEqual:
 			_, _ = tw.Write([]byte("\t!=\t"))
-		case eacl.MatchUnknown:
```

Comment:
Can be left as is not requiring subsequent linter changes.

Reply:
i bet it was previously added to satisfy the linter. Dont see any reason to have empty case

Reply:
Well, you either have this line to satisfy the linter, or another line to do the same thing. If there is no real difference, whatever we already have wins to me. We just don't gain anything with the change.

Reply:
> have this line to satisfy the linter

having it dont fix the linter, otherwise i wouldnt touch this code place at all

Reply:
i strongly disagree with dropping cases and using `nolint` for them. in general, it shows that you skip smth intentionally and does not require to think about it if you are looking for a bug. not critical though

Reply:
Likely can be solved with `default`, btw, one line, a bit more meaning than `//nolint`.

Reply:
so think i: https://github.com/nspcc-dev/neofs-node/pull/2742#discussion_r1507113032

Reply:
https://github.com/nspcc-dev/go-ordered-json/pull/12/commits/c406b37478706212bddd8de3434a44e76a08ef59

The other option of course is to not use `exhaustive` like in https://github.com/nspcc-dev/neo-go/blob/24041451335d2f6ff64b8e334401d2dff7c3b211/.golangci.yml#L44

Reply:
i don't realize the benefit of `exhaustive`. It helped me 0 times, but swears all the time

Reply:
it helped me with updating for sure. if you add more enums, the linter does not allow you to skip places where enums are used, you _have_ to look at them and be sure they are ok or extend switches